validations/validators.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import config
+import logging
+from scripts.factory.pre_flight_check import PreFlightValidationsCall
+from scripts.factory.cluster_readiness_check import ClusterValidationsCall
+from scripts.factory.cortx_check import CortxValidationsCall
+from scripts.factory.unboxing_readiness_check import UnboxingValidationsCall
+
+logger = logging.getLogger(__name__)
+
+
+class Validators():
+    """Validation of all checks for given request."""
+
+    @staticmethod  # noqa: C901
+    def factory_checks(args):
+        if args.postcheck:
+            check_list = config.FACTORY_POST_CHECK
+        elif args.precheck:
+            check_list = config.FACTORY_PRE_CHECK
+        elif args.replacenode:
+            check_list = config.REPLACE_NODE_CHECK
+        elif args.fwupdate:
+            check_list = config.FW_UPDATE_CHECK
+        elif args.unboxing:
+            check_list = config.UNBOXING_CHECK
```

Comment:
We need to separate this into something like '--preunboxing' and '--postunboxing' as we have pre and post checks for unboxing. Please take a look at validations/validators.py and validations/config.py in PR#655.